.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,19 @@
 <!-- Please confirm that your work is based on the correct branch. -->
-<!-- Bug fixes should based on *maint*. -->
-<!-- New features should based on *master*. -->
+<!-- Bug fixes should based on *trunk-patch*. -->
+<!-- Backwards compatible new features should based on *trunk-minor*. -->
+<!-- Incompatible API changes should be based on *trunk-major*. -->
 
 ## Description
 
 <!-- Describe your changes in detail. -->
 
+<!-- Replace ??? with the issue number that this pull request resolves. -->
+Resolves #???
+
```

Comment:
I think this fits better in the motivation/context section where it previously existed. When I am reviewing a pull request, to get background I usually look at the issues the PR fixes.

Reply:
ok